app/containers/IntentService.jsx

```diff
@@ -0,0 +1,102 @@
+/** @jsx h */
+import { h, Component } from 'preact'
+
+import Loading from '../components/Loading'
+
+export default class IntentService extends Component {
+  constructor (props, context) {
+    super(props, context)
+    this.store = context.store
+
+    const {window} = props
```

Comment:
May be a silly question but I do not understand the purpose of this : ```const {window} = props```. It looks like a bad thing.